src/EnergyPlus/VariableSpeedCoils.cc

```diff
@@ -7699,9 +7693,6 @@ namespace VariableSpeedCoils {
         state.dataVariableSpeedCoils->QSource *= PartLoadRatio;
         QWasteHeat *= PartLoadRatio;
 
-        //  Add power to global variable so power can be summed by parent object
-        state.dataHVACGlobal->DXElecHeatingPower = state.dataVariableSpeedCoils->Winput;
-
```

Comment:
Moved `state.dataHVACGlobal->DXElecHeatingPower` to `UpdateVarSpeedCoil()` function such the latest iteration value is used.